src/renderer/component/video/view.jsx

```diff
@@ -55,9 +55,13 @@ class Video extends React.PureComponent {
       contentType,
       changeVolume,
       volume,
+      claim,
       uri,
-      videoPause,
-      setVideoPause,
+      doPlay,
+      doPause,
+      savePosition,
+      mediaPaused,
+      mediaPosition,
```

Comment:
The fact that `Video` is receiving so many properties simply to pass them on to `VideoPlayer` makes me think `VideoPlayer` should just be a first-class component. Although perhaps the names should be clarified if this is the case.

Reply:
I was also thinking it should be its own component. Maybe as a separate issue along with displaying a paused video instead of the image wrapper when position has a value?

Reply:
I agree. #825 can work as a reference as the media component has been separated there. Also I think that Video and VideoPlayer names are very specific, we use the same component to render all other types of media as well, so IMO the variables and components should be refactored to reflect the same(also in the PR).
Thoughts?